src/twisted/mail/relaymanager.py

```diff
@@ -841,11 +845,11 @@ def _ebExchange(self, failure, factory, domain):
         @type domain: L{bytes}
         @param domain: A domain.
         """
-        log.err('Error setting up managed relay factory for ' + domain)
+        log.err(b'Error setting up managed relay factory for ' + domain)
         log.err(failure)
 
         def setWaiting(queue, messages):
-            map(queue.setWaiting, messages)
+            [queue.setWaiting(msg) for msg in messages]
```

Comment:
There's no need to collect the results in a list; perhaps an ordinary for-loop would be clearer here